frontend/src/components/phones/onboarding/RealPhoneSetup.module.scss

```diff
@@ -274,3 +260,162 @@
     text-align: center;
   }
 }
+
+// 3rd party custom input for verifying true phone number
+/* stylelint-disable selector-class-pattern */
+.step :global {
```

Comment:
Are all these styles just copy-pasted from the component? If so, could we not just do, like mentioned in the README:

```js
import "react-phone-number-input/style.css";
```

?